src/rust/plugin-registry/src/error.rs

```diff
@@ -65,7 +65,9 @@ impl From<PluginRegistryServiceError> for Status {
             Error::NomadJobAllocationError => {
                 Status::internal("Unable to allocate Nomad job - it may be out of resources.")
             }
-            Error::ArtifactTooLargeError(msg) => Status::invalid_argument(msg),
+            Error::StreamInputError(_) => {
```

Comment:
Seems like we could format the value in the enum into the message? dunno if it would help

Reply:
This is the over-the-wire message, which we usually want to anonymize. In this case it might make sense